### DISCUSSION TITLE
后端只有二进制文件了，没办法在vercel上部署。

### DISCUSSION THREAD

**Discussion by love768952693** [metatube-community/jellyfin-plugin-metatube#183] [Q&A]
大佬们，后端现在只发布二进制文件了，想叉一个部署在vercel，有解决方案不。

**Comment by xjasonlyu** [comment]
vercel 上本来就不方便部署，只能执行10s，基本没啥用。

**Reply by love768952693** [reply]
感谢大佬秒回，组织提供的免费后端后续用的人多了，肯定不行，所以想自己搭个，学习海外文化太难了，Heroku 、mogenius收费了，Koyeb验证卡发现中国人直接拒，自己的梯子带绿化功能。。。。想搭个免费的环境好难啊。
